src/render/text.rs

```diff
@@ -367,17 +376,36 @@ pub fn render(
     Ok(())
 }
 
+/// Creates the base font name for a font with a specific glyph subset.
+/// Consists of a subset tag and the PostScript name of the font.
+///
+/// Returns a string of length maximum 116, so that even with `-Identity-H`
+/// added it does not exceed the maximum PDF/A name length of 127.
```

Comment:
Can font names contain binary characters, in theory? If so, they will be escaped in `pdf-writer`.

Reply:
My reading of the spec is that the 127 bytes limit is about the resulting name and not the encoding as PDF bytes. Would need to check with verapdf to be sure.

Reply:
I checked and it is indeed fine. `Name("(".repeat(100).as_bytes())` is accepted by verapdf.